examples/dex/crd.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  creationTimestamp: 2017-09-13T19:56:28Z
+  name: authcodes.dex.coreos.com
```

Comment:
You can leave only the `name`, all the other info is not necessary and could cause some issues.

Reply:
⌘+C, ⌘+V